frontends/web/src/routes/account/send/send.tsx

```diff
@@ -501,6 +501,10 @@ class Send extends Component<Props, State> {
       address = url.pathname;
       if (this.isBitcoinBased()) {
         amount = url.searchParams.get('amount') || '';
+        if (amount && this.state.btcUnit === 'sat') {
+          // convert expected amount in BTC to sat
+          amount = `${Number(amount) * 100_000_000}`;
```

Comment:
I considered if this needs to be in try/catch if the qr code has evil data in the amount field, but afaik Number will just return NaN, so worstcase this will be a string with 'NaN' that then is shown as "Invalid amount" error

![Screen Shot 2023-02-16 at 15 16 49](https://user-images.githubusercontent.com/546900/219391085-0fc50c4d-ba66-42eb-9636-6880f27531ec.png)